src/main/java/com/openlattice/postgres/PostgresTable.java

```diff
@@ -165,6 +165,34 @@
                             CATEGORY,
                             SHARDS );
     //.setUnique( NAMESPACE, NAME );
+    public static final PostgresTableDefinition        ENTITY_TYPE_COLLECTIONS      =
+            new PostgresTableDefinition( "entity_type_collections" )
+                    .addColumns(
+                            ID,
+                            NAMESPACE,
+                            NAME,
+                            TITLE,
+                            DESCRIPTION,
+                            SCHEMAS,
+                            TEMPLATE );
+    public static final PostgresTableDefinition        ENTITY_SET_COLLECTIONS       =
+            new PostgresTableDefinition( "entity_set_collections" )
+                    .addColumns(
+                            ID,
+                            NAME,
+                            TITLE,
+                            DESCRIPTION,
+                            CONTACTS,
+                            ENTITY_TYPE_COLLECTION_ID,
+                            ORGANIZATION_ID );
+    public static final PostgresTableDefinition        ENTITY_SET_COLLECTION_CONFIG =
+            new PostgresTableDefinition( "entity_set_collection_config" )
+                    .addColumns(
+                            ENTITY_SET_COLLECTION_ID,
+                            TEMPLATE_TYPE_ID,
+                            ENTITY_SET_ID )
+                    .primaryKey( ENTITY_SET_COLLECTION_ID, TEMPLATE_TYPE_ID )
+                    .setUnique( ENTITY_SET_COLLECTION_ID, TEMPLATE_TYPE_ID );
```

Comment:
I don't think it's necessary to set these unique, since they are the primary key

Reply:
+1 this won't work on citus.

Reply:
Actually, it will be fine on citus matter since these are in a distributed table.

Reply:
this was removed and isn't part of this PR anymore